src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy

```diff
@@ -46,7 +47,7 @@ class GCloudConfigurable {
     }
 
     void args(Map<String, ?> args) {
```

Comment:
Hrmm, I think it was to enable sending in Map<String, String> or Map<String, Boolean> or whatever as a parameter list. Bad idea?